.github/workflows/ci.yml

```
@@ -13,4 +13,21 @@ jobs:
     name: CI
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
-    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    uses: maxime-rainville/gha-ci/.github/workflows/ci.yml@v1
```

```suggestion
    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```

Need to switch this back before merging.